VEHICLE/AttachVehicleToCargobob.md

```diff
@@ -5,18 +5,47 @@ ns: VEHICLE
 
 ```c
 // 0x4127F1D84E347769 0x607DC9D5
-void ATTACH_VEHICLE_TO_CARGOBOB(Vehicle vehicle, Vehicle cargobob, int p2, float x, float y, float z);
-```
-
-```
-p2 is unknown and is always -1 in the script natives.
+void ATTACH_VEHICLE_TO_CARGOBOB(Vehicle cargobob, Vehicle vehicle, int vehicleBoneIndex, float x, float y, float z);
 ```
 
 ## Parameters
-* **vehicle**: The vehicle which should be attached
 * **cargobob**: The cargobob
-* **p2**:
-* **x**:
-* **y**:
-* **z**:
+* **vehicle**: The vehicle which should be attached
```

Comment:
nit pick here, "The vehicle that will be attached" (feel free to ignore this though, personal preference)